src/components/textarea/textarea.component.js

```diff
@@ -102,6 +98,16 @@ const Textarea = ({
     fieldHelp,
   });
 
+  const [maxLength, characterCount] = useCharacterCount(
+    value,
+    // TODO: Can be removed after the characterLimit type is changed to number
```

Comment:
We'll be holding off on making the breaking change until we can batch it with some other ones

Reply:
Myself and @mkrds discussed privately and decided to hold off on making the breaking change, we'll be holding off on making the breaking change until we can batch it with some other ones

Reply:
Nice one. The description of the PR will need changing as it has `BREAKING CHANGE: characterLimit prop in both Textbox and Textarea is now of type number`. I know none of the commits have the `BREAKING CHANGE` specified but it might be confusing if anyone looks back at the PR and see that in the description.